src/App.css

```diff
@@ -8,46 +32,132 @@
   align-items: center;
 }
 
+.navbar {
+  width: 100%;
+  display: flex;
```

Comment:
- [ ] This requirement I am marking as optional because I didn't want to return you because of styling, but please implement it since you are repeating yourself with `display: flex` 5 times and that is bad CSS practice. I would suggest you create a class name that has a display: flex and use it on multiple jsx elements

Reply:
Thank you for that suggestion, the change has been made 👍